packages/manager/modules/pci/src/projects/project/streams/stream/stream.html

```diff
@@ -0,0 +1,101 @@
+<oui-page-header data-heading="{{:: $ctrl.stream.name }}">
+    <oui-header-tabs>
+        <oui-header-tabs-item
+            data-text="{{:: 'pci_projects_project_streams_stream_dashboard_title' | translate }}"
+            data-href="{{:: $ctrl.streamLink }}"
+            data-active="$ctrl.currentActiveLink() === $ctrl.streamLink">
+        </oui-header-tabs-item>
+        <oui-header-tabs-item
+            data-text="{{:: 'pci_projects_project_streams_stream_subscriptions_title' | translate }}"
+            data-href="{{:: $ctrl.subscriptionsLink }}"
+            data-active="$ctrl.currentActiveLink() === $ctrl.subscriptionsLink">
+        </oui-header-tabs-item>
+    </oui-header-tabs>
+
+</oui-page-header>
+
+<div class="mt-4">
+    <cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+</div>
+
+<div class="py-4" data-ui-view>
+    <div class="row">
+        <div class="col-md-4">
+            <oui-tile data-heading="{{:: 'pci_projects_project_streams_stream_manage_title' | translate }}">
+                <oui-tile-button data-on-click="$ctrl.editBacklogRetention()"
+                    data-ng-if="$ctrl.stream.isPersistent()">
```

Comment:
`ng-if` could be grouped with the one below

Reply:
More easy to enable/disable actions one by one.